oe_bootstrap_theme.libraries.yml

```
@@ -18,6 +18,10 @@ drupal.progress:
   js:
     assets/js/progress.js: {}
 
+drupal.inpage_navigation:
```

the convention is that you use drupal.something when you are overwriting a drupal core library (Ex drupal.progress), here it is not the case
```suggestion
inpage_navigation:
```